wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/InstantCommand.java

```diff
@@ -0,0 +1,22 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.command;
+
+public class InstantCommand extends Command {
+  public InstantCommand() {
+    super();
```

Comment:
We don't know much about Java, especially how its inheritance works. We just used the [patch](https://paste.unixcube.org/t/5f8a82) you linked from pastebin. If you think it can be better, tell us what to change and we'll do it.